doc.go

```diff
@@ -0,0 +1,52 @@
+// Package shrub provides a simple, low-overhead interface for
+// generating Evergreen project configurations.
+//
+// For most use cases, you can start with a Configuration struct and add data
+// such as new tasks, task groups, build variants, and functions to that
+// Configuration using the provided setters. For example:
+//
+//	conf := &Configuration{}
```

Comment:
should we move this to line 21?

Reply:
or maybe add a line saying something like start by initializing a configuration?